mmv1/templates/terraform/examples/bigquerydatatransfer_config_pub_sub.tf.tmpl

```diff
@@ -0,0 +1,44 @@
+data "google_project" "project" {
+}
+
+resource "google_project_iam_member" "permissions" {
```

Comment:
We cannot use this resource to grant permissions to a project-wide service account because other tests running in parallel will lose permissions when the iam member is cleaned up.

I recommend putting this test in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/bigquerydatatransfer/resource_bigquery_data_transfer_config_test.go instead and using `exclude_test: true` in the example yaml for consistency with the rest of the bigquery data transfer config resource.